pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg` and `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Print an image or animated GIF to the terminal:
```

Comment:
All the tldr examples are for the terminal, should the user use ctrl + C or q to exit?
```suggestion
- Print an image or animated GIF:
```

Reply:
It's creating an image out of text characters inside the terminal. And I generally think of "print to the terminal" as a sort of phrase, since just using "print" might make you think of a GUI application or an actual printer.

I already pretty much said this in the description though, so I'd understand if it seems redundant.